.pre-commit-config.yaml

```diff
@@ -4,6 +4,14 @@ repos:
     hooks:
       - id: pyupgrade
         args: [--py39-plus]
+  - repo: https://github.com/PyCQA/autoflake
+    rev: v1.7.7
+    hooks:
+      - id: autoflake
+        args:
+          - --in-place
+          - --remove-duplicate-keys
```

Comment:
If this changes

```python
{
  "hello": 1,
  "hello": 2
}
```

to 

```python
{
  "hello": 2
}
```

we shouldn't enable it. The odds are very high that this is a bug, and not sloppiness. The user should decide which value is kept (or fix the key)

Reply:
That's what it does

I'll assume there's a `pylint` or other rule to make sure double dict keys fail down the line.